src/main/java/frc/robot/commands/LimelightAlign.java

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import frc.robot.subsystems.LimelightSubsystem;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class LimelightAlign extends CommandBase {
+  private LimelightSubsystem m_LimelightSubsystem;
+
+  /** Creates a new ShooterPrimary. */
+  public LimelightAlign(LimelightSubsystem LimelightSubsystem) {
+    addRequirements(LimelightSubsystem);
+    m_LimelightSubsystem = LimelightSubsystem;
+
+    // Use addRequirements() here to declare subsystem dependencies.
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    m_LimelightSubsystem.setMotors();
+  }
+
+  // Called once the command ends or is interrupted.
+  @Override
+  public void end(boolean interrupted) {
+    m_LimelightSubsystem.setMotors();
```

Comment:
Does setMotors need to be called again here? Seems like the motors should have already stopped. 

Should also explicitly stop the motors here in case the command is interrupted/canceled before it's finished.